rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/CommonRestInterface.java

```diff
@@ -121,7 +121,7 @@ void logout(@HeaderParam("sessionid")
     /**
      * Get a UserData object associated to a session.
      *
-     * <br/> NOTE: <br/>
+     * <br> NOTE: <br>
```

Comment:
Shouldn't it be <br /> to skip line ?

Reply:
it doesn't work like this as this api is also used to generate the rest documentation, we need to use <br>